Corona-Warn-App/src/main/res/layout/fragment_trace_location_onboarding.xml

```diff
@@ -0,0 +1,260 @@
+                android:layout_marginTop="24dp"
+                android:src="@drawable/checkin_warning"
+                android:layout_marginStart="24dp"
+                app:layout_constraintTop_toBottomOf="@+id/check_in_onboarding_subtitle"
+                app:layout_constraintStart_toStartOf="parent" />
+
+            <TextView
+                android:id="@+id/check_in_onboarding_warning"
+                style="@style/subtitle"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="16dp"
+                android:text="@string/trace_location_onboarding_body_warning"
+                app:layout_constraintTop_toTopOf="@id/check_in_warning_image"
+                app:layout_constraintStart_toEndOf="@id/check_in_warning_image"
+                app:layout_constraintEnd_toEndOf="parent" />
+
+            <ImageView
+                android:id="@+id/check_in_stay_image"
+                android:layout_width="32dp"
+                android:layout_height="32dp"
+                android:layout_marginTop="24dp"
+                android:src="@drawable/checkin_stay"
+                android:layout_marginStart="24dp"
+                app:layout_constraintTop_toBottomOf="@+id/check_in_onboarding_warning"
+                app:layout_constraintStart_toStartOf="parent" />
+
+            <TextView
+                android:id="@+id/check_in_onboarding_stay"
+                style="@style/subtitle"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="16dp"
+                android:text="@string/trace_location_onboarding_body_stay"
+                app:layout_constraintTop_toTopOf="@id/check_in_stay_image"
+                app:layout_constraintStart_toEndOf="@id/check_in_warning_image"
+                app:layout_constraintEnd_toEndOf="parent" />
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/check_in_onboarding_card_container"
+                style="@style/cardTracing"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="24dp"
+                android:layout_marginHorizontal="24dp"
+                android:background="#F5F5F5"
+                android:orientation="vertical"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/check_in_onboarding_stay">
+
+                <TextView
+                    android:id="@+id/check_in_onboarding_card_title"
+                    style="@style/headline6"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/trace_location_onboarding_card_title"
```

Comment:
All strings in this constraint layout need to be moved to the legal strings files.

Reply:
thanks for the hint